README.md

```diff
@@ -39,40 +39,44 @@ As an abstraction, this tool allows for greater consistency and maintainability
 
 ## Overview
 
-At its core, RESTful React exposes a component, called `Get`. This component retrieves data, either on mount or later, and then handles error states, caching, loading states, and other cases for you. As such, you simply get a component that _gets stuff_ and then does stuff with it. Here's a quick overview what it looks like.
+At its core, `restful-react` exposes a [hook](https://reactjs.org/docs/hooks-intro.html), called `useGet`. This component retrieves data, either on mount or later, and then handles error states, loading states, and other cases for you. As such, you get a component that _gets stuff_ and then does stuff with it. Here's a quick overview what it looks like.
 
-[![Edit Restful React demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/30n66z45mq)
+[![Edit restful-react demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/restful-react-demos-vjets)
 
 ```jsx
 import React from "react";
-import Get from "restful-react";
+import { useGet } from "restful-react";
 
-const MyComponent = () => (
-  <Get path="https://dog.ceo/api/breeds/image/random">
-    {randomDogImage => <img alt="Here's a good boye!" src={randomDogImage && randomDogImage.message} />}
-  </Get>
-);
+const MyComponent = () => {
+  const { data: randomDogImage } = useGet({
+    path: "https://dog.ceo/api/breeds/image/random",
+  });
+
+  return <img alt="Here's a good boye!" src={randomDogImage && randomDogImage.message} />;
+};
 
 export default MyComponent;
 ```
 
 ## Getting Started
 
-To install and use this library, simply `yarn add restful-react`, or `npm i restful-react --save` and you should be good to go. Don't forget to `import Get from "restful-react"` or similar wherever you need it!
+To install and use this library, install it by running `yarn add restful-react`, or `npm i restful-react --save` and you should be good to go. Don't forget to `import { useGet } from "restful-react"` or similar wherever you need it!
 
 ## Features
 
-Restful React ships with the following features that we think might be useful.
+`restful-react` ships with the following features that we think might be useful.
 
 ### Global Configuration
 
-API endpoints usually sit alongside a base, global URL. As a convenience, the `RestfulProvider` allows top-level configuration of your requests, that are then passed down the React tree to `Get` components.
+REST API endpoints usually sit alongside a base, global URL. As a convenience, the `RestfulProvider` allows top-level configuration of your requests, that are then passed down the React tree to `useGet` hooks.
 
 Consider,
 
-[![Edit Restful React demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/30n66z45mq)
+[![Edit restful-react demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/restful-react-demos-vjets)
```

Comment:
Same demo link as the the one above. is it intentional?

Reply:
Yes.